examples/visualization/brain.py

```diff
@@ -0,0 +1,106 @@
+"""
+.. _ex-brain:
+
+===============================
+Plotting with ``mne.viz.Brain``
+===============================
+
+In this example, we'll show how to use :class:`mne.viz.Brain`.
+"""
+# Author: Alex Rockhill <aprockhill@mailbox.org>
+#
+# License: BSD-3-Clause
+
+# %%
+# Plot a brain image
+# ------------------
+#
+# First we plot a brain. This requires a directory with MRI? information.
+# MNE provides functionality to automatically retrieve standard head images.
+# In this example we use the ``sample`` data which is data from an example
+# subject being shown auditory and visual stimuli.
+
+import os.path as op
+import matplotlib.pyplot as plt
+
+import mne
+from mne.datasets import sample
+
+print(__doc__)
+
+data_path = sample.data_path()
+subjects_dir = op.join(data_path, 'subjects')
+brain = mne.viz.Brain('sample', subjects_dir=subjects_dir,
+                      alpha=0.5, background='white', cortex='low_contrast')
+
+
+# %%
+# Add source information
+# ----------------------
+#
+# Plot source information.
+sample_dir = op.join(data_path, 'MEG', 'sample')
+evoked = mne.read_evokeds(op.join(sample_dir, 'sample_audvis-ave.fif'))[0]
+trans = mne.read_trans(op.join(sample_dir, 'sample_audvis_raw-trans.fif'))
+stc = mne.read_source_estimate(op.join(sample_dir, 'sample_audvis-meg'))
+stc.crop(0.09, 0.1)
+
+fmin, fmax = stc.data.min(), stc.data.max()
+for i, hemi in enumerate(('lh', 'rh')):
+    hemi_data = stc.data[:len(stc.vertices[i])]
```

Comment:
I doubt this is correct as this will always select the left hemisphere time courses

Reply:
Good catch